README-source.md

```diff
@@ -269,18 +269,19 @@ zapier deprecate 1.0.0 2017-01-01
 
 ## Converting an Existing App
 
-> Warning! This is in a **very** alpha state - the immediate goal is to provide some basic structure to match an existing application. It will not even get close to a working copy of your existing app (yet).
-
-If you have an existing application (nominally named "V2") on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
+If you have an existing Web Build app on https://zapier.com/developer/builder/ you can use it as a template to kickstart your local application.
 
 ```bash
-# the 1234 is from the URL in https://zapier.com/developer/builder/
-zapier convert 1234 .
+# Convert an existing Web Builder app to CLI in my-app directory
+# App ID 1234 is from URL https://zapier.com/developer/builder/
```

Comment:
Give a full url instead. 

`The App ID is a series of numbers near the end of your web builder link: https://zapier.com/developer/builder/app/1234/development`